File: bubble_sort/Python/bubbleSort.py
#Bubble Sort in Python

array = [] #declaration of array


#getting f user inputs and append that values to the array
for x in range(5):
    array.append(eval(input("number " + "")))

n = len(array) #array length
print(array)  #prints initial array (unsorted)


def bubbleSort(array):
    for j in range(n):
        for i in range((n - 1), j, -1):
            if(array[i] < array[i - 1]):
                temp = array[i]
                array[i] = array[i - 1]
                array[i - 1] = temp


bubbleSort(array)
print(array)

File: bubble_sort/Javascript/bubblesort.js

function bubbleSort(values) {
    for (let i = values.length - 1; i >= 0; i--) {
        for(let j = 1; j <= i; j++) {
            if(values[j - 1] > values[j]) {
                let temp = values[j - 1];
                values[j - 1] = values[j];
                values[j] = temp;
            }
        }
    }
    return values;
}

console.log(bubbleSort([94, 22, 12, 7, 390, 3, 78]));
